core/inject.js

```diff
@@ -29,6 +29,8 @@ goog.require('Blockly.utils.userAgent');
 goog.require('Blockly.WorkspaceDragSurfaceSvg');
 goog.require('Blockly.WorkspaceSvg');
 
+goog.requireType('Blockly.utils.Metrics');
```

Comment:
Does this need to be here?

Reply:
Yes, it's used in the getMetrics definition.